skimage/novice/_novice.py

```diff
@@ -326,6 +326,9 @@ def save(self, path):
         path : str
             Path (with file extension) where the picture is saved.
         """
+        if (self.array.ndim >= 3 and self.array.shape[-1] >= 4 and
+                os.path.splitext(path)[-1] in ['.jpg', '.jpeg']):
```

Comment:
To be safe, `os.path.splitext(path)[-1] in ['.jpg', '.jpeg'].lower() in ['.jpg', '.jpeg']`?